### ISSUE TITLE
amp-story: Missing or inaccurate captions for prerecorded content

### ISSUE THREAD

**Issue by TetraLogicalHelpdesk** [ampproject/amphtml#32496]
# Missing or inaccurate captions for prerecorded content

## WCAG Level
Level A

## Priority
Low

## Pages/screens/components affected
* [Video](https://www.gstatic.com/amphtml/stamp/qa/video.html)
* [AMP Story Player - AMP Story 360](https://www.gstatic.com/amphtml/stamp/qa/amp-story-360.html)

## Description
When captions are not available for video/multimedia content that has an audio track, people who cannot hear the audio will find it difficult or impossible to understand the content.

Captions are a text alternative to the audio content of a video or multimedia presentation, synchronised with the original audio track. Typically captions contain the dialogue being spoken (including who is speaking), plus descriptions of other important sounds.

With the exception of the last page in the [Video](https://www.gstatic.com/amphtml/stamp/qa/video.html) test page, none of the videos with meaningful audio (in that same AMP story, as well as in the [AMP Story Player - AMP Story 360](https://www.gstatic.com/amphtml/stamp/qa/amp-story-360.html) sample) provide captions. This issue is marked as "Low" priority, but only because of the sample nature of the videos.

Note that video is used in other tested samples. However, in those cases, the video had no significant/meaningful audio that would necessitate captions.

## User impact
Without accurate captions someone who is deaf or hard of hearing will find it difficult or impossible to understand the video or multimedia content, particularly if it contains spoken dialogue.

Captions are also useful to people in noisy environments where the ambient noise makes it hard to hear audio content, and to people in quiet environments where turning up the volume may disturb other people nearby.

## Required solution
Provide synchronised captions for each video/multimedia content that has meaningful audio (audio that conveys information, rather than just being "decorative", like instrumental mood music or similar), and make sure that the dialogue and all important sound-effects are included.

Closed captions (CC) are the recommended way to provide captions. These are captions that can be displayed on demand, usually by activating a control in the media player interface. Open captions are added directly the video or multimedia content itself and are displayed in all circumstances. While open captions are an acceptable method of meeting this requirement, it is recommended that closed captions are provided, as they offer greater flexibility and the opportunity to be customised to suit the needs of the viewer.

This solution must be applied to all instances of the issue identified within the test sample, then applied to all other instances of the same issue identified throughout the rest of the website.

## Implementation guidance

The last page in [Video](https://www.gstatic.com/amphtml/stamp/qa/video.html) gives an example of a video that shows captions by default.

![The last page in the video-specific AMP story, showing the background video with visible captions displayed](https://user-images.githubusercontent.com/54933393/107244098-91f4b600-6a25-11eb-9077-83bbaa9b47dd.png)

However, this approach may not work in all situations. One potential approach would be to provide an extra general control (similar to the sound on/off and play/pause control) that allows users to explicitly turn closed captions on and off.

## Test procedure(s)
Use these steps to confirm that the solution has been correctly applied to issues identified within the test sample, and to test the rest of the website for instances of the same issue:
1. Check that captions are available for all videos/multimedia content with meaningful audio content that contains spoken dialogue and sound-effects.
2. Check that the captions are an accurate representation of the spoken dialogue and sound-effects in the audio content.

### Definition of done
Complete all of these tasks before closing this issue or indicating it is ready for retest:
* All issues identified within the test sample have been resolved.
* The rest of the website has been tested for the same issue.
* All issues identified throughout the rest of the website have been resolved or filed as new issues.

## Related standards
* [WCAG 2.1 Success Criterion 1.2.2 Captions (prerecorded) (Level A)](https://www.w3.org/TR/WCAG21/#captions-prerecorded)

## More information
* [WebAIM: Captions](https://webaim.org/techniques/captions/)
* [Nomensa: What are captions?](https://www.nomensa.com/blog/2010/what-are-captions)

## Test data
Test date: January 2021

**Comment by gmajoulet** [comment]
Captions have to be provided by the publishers, which we do recommend but can't enforce.
Keeping this ticket open to discuss whether we should have a way to enable/disable captions, when available.
